pages/success-banner.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Head from 'next/head';
+
+class Button extends React.Component {
+  render() {
+    return (
+      <a
+        href={ this.props.href }
+        target="_blank"
+        className="button">
+        { this.props.children }
+      </a>
+    )
+  }
+}
+
+const Banner = () => (
+  <main className="vertical center">
+    <div>
+      <h1>We hope you enjoyed using</h1>
+      <img src="/static/etcher.svg" className="etcher-logo"></img>
```

Comment:
I think you likely want to use the margin bottom from h1 as spacing so it's simpler (less css) to have it outside `h1`. Though no need to close the image tag.